frontend/src/lib/components/LemonModal/LemonModal.scss

```diff
@@ -16,11 +17,13 @@
     &.ReactModal__Overlay--after-open {
         background-color: rgba(0, 0, 0, 0.2);
         backdrop-filter: blur(var(--modal-backdrop-blur));
+        -webkit-backdrop-filter: blur(var(--modal-backdrop-blur));
```

Comment:
Safari amazingly needs this still

Reply:
Interesting, don't we have autoprefixer for this?

Reply:
I wondered this as well but apparently not as this problem is in production too